app/views.py

```diff
@@ -3,12 +3,18 @@
 import re
 from json import loads
 
-from app import app, models, sql_session, viewsModel
```

Comment:
This makes so much sense.  I totally see how importing specific classes and functions from their submodules would make it much easier to keep track of all modules affected by a change to a class or function and thus easier to maintain your codebase.  

I do have a question about minimizing coupling as a rule though:  as I understand it the MVC pattern is useful because it separates database queries and templates from the main code, thereby allowing you to reuse queries and templates in different parts of the codebase.  For that reason, I'm hesitant to include all of the views_model functions in the controller (confusingly named views.py).  However, I do agree that my views_model could be better organized.  Perhaps I could divide the views_model into separate models based on objects like playlist_model, video_model, etc.  I know that wouldn't reduce the coupling, but what do you think?

Thanks again for all your feedback-- it has been incredibly helpful!